.travis.yml

```diff
@@ -12,11 +12,12 @@ before_install:
 - sed -i -e s/dss.data.humancellatlas.org/dss.$DSS_TEST_STAGE.data.humancellatlas.org/ hca/default_config.json
 install:
 - pip install -r requirements-dev.txt
-- wget -O regex.json https://raw.githubusercontent.com/HumanCellAtlas/dcplib/master/components/trufflehog_regex_patterns.json
-- trufflehog --regex --rules regex.json --entropy=False https://github.com/HumanCellAtlas/data-store.git
-- rm regex.json
+- pip install trufflehog
 - make install
 script:
+- wget -O regex.json https://raw.githubusercontent.com/HumanCellAtlas/dcplib/master/components/trufflehog_regex_patterns.json
+- trufflehog --regex --rules regex.json --entropy=False https://github.com/HumanCellAtlas/dcp-cli.git
```

Comment:
Consider replacing the hardcoded repo URL with something like `$(git remote get-url origin)`.

Reply:
That seems much more robust.  Thanks, I'll add it.